npc/re/mobs/int_land.txt

```diff
@@ -0,0 +1,13 @@
+//===== rAthena Script =======================================
+//= Beginner Area Monster Spawn Script
+//===== By: ==================================================
+//= Jenkijo
+//===== Current Version: =====================================
+//= 1.0
+//===== Compatible With: =====================================
+//= rAthena Project
+//===== Additional Comments: =================================
+//= 1.0 First version.
+//============================================================
+
+int_land,0,0	monster	Poring	1002,25,5000
```

Comment:
From playing on kRO again:
These are either not normal porings or they have no_exp & no_drop mapflags on these maps.
They only drop Lumber and you get it DIRECTLY into your inventory. The only thing they dropped for me were novice clothes that were probably dropped by other players before and the porings probably looted them.